### PR TITLE
Bump PHP/Symfony versions

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

**Comment by XWB** [comment]
@stof Do you perhaps have any idea why Doctrine Annotations cannot be loaded on >= Symfony 3, even when the library is installed via composer?

> {"error":{"code":500,"message":"Internal Server Error","exception":[{"message":"[Semantical Error] The annotation \"@Doctrine\\ORM\\Mapping\\Entity\" in class FOS\\CommentBundle\\Tests\\Functional\\Bundle\\CommentBundle\\Entity\\Thread does not exist, or could not be auto-loaded.","class":"Doctrine\\Common\\Annotations\\AnnotationException","trace":[{"namespace":"","short_class":"","class":"","type":"","function":"","file":"\/FOSCommentBundle\/vendor\/doctrine\/annotations\/lib\/D"...

**Comment by XWB** [comment]
@stof I changed everything but Travis still fails on the following error:

> {"error":{"code":500,"message":"Internal Server Error","exception":[{"message":"[Semantical Error] The annotation \"@Doctrine\\ORM\\Mapping\\Entity\" in class FOS\\CommentBundle\\Tests\\Functional\\Bundle\\CommentBundle\\Entity\\Thread does not exist, or could not be auto-loaded.","class":"Doctrine\\Common\\Annotations\\AnnotationException","trace":[{"namespace":"","short_class":"","class":"","type":"","function":"","file":"\/FOSCommentBundle\/vendor\/doctrine\/annotations\/lib\/D"...

Any idea?

File: composer.json [hunk 1]
```diff
@@ -24,44 +24,51 @@
...
+        "phpunit/phpunit": "~4.8|~5.0"
     },
     "extra": {
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
     },
     "autoload": {
-        "psr-0": { "FOS\\CommentBundle": "" }
+        "psr-4": { "FOS\\CommentBundle\\": "" },
```
Comment (stof) [inline composer.json]: this change is wrong, as you forgot to remove the ``target-dir``, making the composer.json invalid

Comment (XWB) [inline composer.json]: I pushed a new commit to the master branch.

**Pull request merged**